locations/spiders/groszek_pl.py

```diff
@@ -14,6 +14,8 @@ def parse(self, response, **kwargs):
             item = DictParser.parse(feature)
             item["ref"] = feature["idCRM"]
             item["housenumber"] = feature["number"]
+            if isinstance(item["housenumber"], str) and item["housenumber"].lower() == "b/n":
+                item["housenumber"] = None
```

Comment:
Instead of checking for type, coerce the house number from the data source into a string:

```suggestion
            item["housenumber"] = str(feature["number"])
            if item["housenumber"].lower() == "b/n":
                item["housenumber"] = None
```